Automattic/newspack-newsletters
# Assorted bugfixes

### All Submissions:






### Changes proposed in this Pull Request:



Closes #300
Closes #305

### How to test the changes in this Pull Request:

1. On `master`
2. Create two ads, one published and one draft
3. Create a newsletter, set custom fonts and background color in the Styling panel
4. Send a test email, observe that the draft ad is included in the email, the custom fonts are not applied, and that when the newsletter is viewed as a WP page, the custom styles are not applied
5. Switch to this branch, re-send the email
6. Observe the draft ad is not included, the custom styles are applied both in the email and in the WP page
7. Create a new newsletter, using a template with some headers. 
8. Send a test email, observe the headers have the default Arial font, while the body has the default Georgia font

### Other information:
